src/EPR.Payment.Facade/Services/Payments/OnlinePaymentsService.cs

```diff
@@ -193,56 +194,56 @@ private async Task<GovPayResponseDto> InitiateGovPayPaymentAsync(GovPayRequestDt
             return govPayResponse;
         }
 
-        private async Task UpdatePaymentStatusAsync(Guid externalPaymentId, PaymentRequestDto request, string paymentId, CancellationToken cancellationToken)
+        private async Task UpdateOnlinePaymentStatusAsync(Guid externalPaymentId, OnlinePaymentRequestDto request, string paymentId, CancellationToken cancellationToken)
         {
-            var updateRequest = _mapper.Map<UpdatePaymentRequestDto>(request);
+            var updateRequest = _mapper.Map<UpdateOnlinePaymentRequestDto>(request);
             updateRequest.ExternalPaymentId = externalPaymentId;
             updateRequest.GovPayPaymentId = paymentId;
             updateRequest.Status = PaymentStatus.InProgress;
 
             try
             {
-                await _httpPaymentsService.UpdatePaymentAsync(externalPaymentId, updateRequest, cancellationToken);
+                await _httpOnlinePaymentsService.UpdateOnlinePaymentAsync(externalPaymentId, updateRequest, cancellationToken);
             }
             catch (ValidationException ex)
             {
-                _logger.LogError(ex, LogMessages.ValidationErrorUpdatingPayment);
+                _logger.LogError(ex, LogMessages.ValidationErrorUpdatingOnlinePayment);
                 throw new ServiceException(ex.Message, ex);
             }
             catch (Exception ex)
             {
-                _logger.LogError(ex, LogMessages.UnexpectedErrorUpdatingPayment);
-                throw new ServiceException(ExceptionMessages.UnexpectedErrorUpdatingPayment, ex);
+                _logger.LogError(ex, LogMessages.UnexpectedErrorUpdatingOnlinePayment);
+                throw new ServiceException(ExceptionMessages.UnexpectedErrorUpdatingOnlinePayment, ex);
             }
         }
 
-        private static PaymentResponseDto CreatePaymentResponse(GovPayResponseDto govPayResponse)
+        private OnlinePaymentResponseDto CreateOnlinePaymentResponse(GovPayResponseDto govPayResponse)
```

Comment:
sonarqube: Member 'CreateOnlinePaymentResponse' does not access instance data and can be marked as static

Reply:
done